src/CaptureServiceBase/include/CaptureServiceBase/CaptureServiceBase.h

```diff
@@ -33,11 +33,11 @@ class CaptureServiceBase {
     kGuestOrcConnectionFailure,
   };
 
- protected:
```

Comment:
Why this change? Isn't `InitializeCapture` still only used by the subclasses?

Reply:
Previously we just need to call `DoCapture` from the CloudCollector. But now we move the `InitializeCapture` out from `DoCapture` and we will implement the similar logic in `LinuxCaptureService::Capture` for the CloudCollector.